docs/synthetics/http-monitor.md

```diff
@@ -24,6 +24,13 @@ Optionally, the header, query param, and cookie values can be marked as hidden,
 
 Max length of the header, query param, and cookie name and value are 512 and 1024 characters respectively. A maximum of 100 rows for each of the headers, query params, and cookies is allowed.
 
+By default HTTP monitor adds below headers for all the requests sent from the agent:
```

Comment:
By default HTTP monitor adds below headers for all the requests sent from the agent:
=>
By default, the HTTP monitor adds the headers below for all requests sent from the agent: